zjit/src/backend/lir.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::double_must_use)]
```

Comment:
I think the `#[must_use]` was just left-over from when the return type was an `Option`. We can fix it by removing the `#[must_use]`.